engine/src/test/groovy/graphql/nadel/e2e/NadelE2EDynamicServiceResolutionTest.groovy

```diff
@@ -262,4 +262,101 @@ class NadelE2EDynamicServiceResolutionTest extends Specification {
         result.errors.size() == 1
         result.errors.get(0).message == "Could not resolve service for field: /commit"
     }
+
+    def "handles multiple inline fragments"() {
```

Comment:
I actually think these tests should be not be e2e test at all, just use the new yml test style

Reply:
They're all replicated in the yml format. I'll just delete this class.